domains/iusearchbtw.json

```
@@ -0,0 +1,9 @@
+{
+  "subdomain": "iusearchbtw",
+  "owner": {
+    "email": "uksningaken@gmail.com"
+  },
```

Missing username field
```suggestion
  "owner": {
  	"username": "iamnotmega",
    "email": "uksningaken@gmail.com"
  },
```